recipes/natural_language_processing/chatbot/Makefile

```diff
@@ -21,13 +21,13 @@ quadlet:
 	    -e "s|APPIMAGE|${APPIMAGE}|g" \
 	    -e "s|MODELIMAGE|${MODELIMAGE}|g" \
 	    quadlet/${APP}.image \
-	    > build/${APP}.image
+	    > bootc/build/${APP}.image
```

Comment:
Why not leave this at top level build directory and just change bootc/Containerfile to grab from build rather then quadlet?

I don't think make quadlet should be tied to bootc.

Reply:
updated